sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -122,17 +115,15 @@ public virtual bool IsClosed
                 Argument.AssertNotNullOrWhiteSpace(entityPath, nameof(entityPath));
                 connection.ThrowIfClosed();
 
-                options = options?.Clone() ?? new ServiceBusSenderOptions();
+                options = options ?? new ServiceBusSenderOptions();
```

Comment:
Good catch;  looks like `ServiceBusClient` doesn't even allow options to be set.   Since it appears to be responsible for creating an empty instance is the null coalesce even needed?     You may want to add a comment to document the assumption as well.  Since the receiver still needs the clone, I thought this was a mistake at first glance.

Reply:
yeah I suppose it is not needed, but I left it this way to minimize churn for when we eventually enable.